packages/create-svelte/bin.js

```diff
@@ -75,8 +76,16 @@ async function main() {
 					message: 'Add type checking?',
 					initial: false,
 					choices: [
-						{ title: 'Type-checked JavaScript', value: 'checkjs' },
-						{ title: 'TypeScript', value: 'typescript' },
+						{
+							title: 'JavaScript with checkJs',
```

Comment:
I prefer `Type-checked JavaScript` as title.

Reply:
Tbh, I'm not all that happy with either, as the author of the issue pointed out "Isn't typescript supposed to be type checked javascript", and I agree with that. And 'Javascript with checkJs' probably just confuses people who don't know what checkJs is even more.

Lemme try something else.

Reply:
yes, `checkJS` is really confusing for beginners... actually I think TypeScript is TypeScript, another form of type checked JavaScript, but type checked JavaScript is also JS with JSDocs... so I think name `Type-Checked JavaScript` is fine for me.

Reply:
I'm fine with the titles that are confusing. As long as the description is deconfusing.  

The description for Typescript could be:
 `A superset of JavaScript that adds optional static typing to the language. (for more info see https://www.typescriptlang.org)`

For the `Type-Checked JavaScript` option I'm still not sure what it means haha.
Is this the checkJS you are talking about? https://www.typescriptlang.org/tsconfig#checkJs

I've also found [this page] (https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html) which also describes other forms of js type checking besides the JSDoc. 
Am I correct in concluding that the JSDoc-based-type-checks are only a subset of the type checking done?
Maybe the description can refer to that page.

But having read all this information, the best of these option for a new project seems to be Typescript, right?
What is the situations where you want to have the js with typechecking?

Reply:
> Is this the checkJS you are talking about? [typescriptlang.org/tsconfig#checkJs](https://www.typescriptlang.org/tsconfig#checkJs)

Yes it was, although now I've omitted the descriptions so the preview above is outdated.

> The description for Typescript could be:
 A superset of JavaScript that adds optional static typing to the language. (for more info see https://www.typescriptlang.org/)

That might be a bit too lengthy. If someone doesn't know then it's a quick search away, it's a different language so that'll provide a lot more information than we ever could in a CLI prompt.

> What is the situations where you want to have the js with typechecking?
If that situtation almost never happens wouldn't it be better to have typescript as the recommended option?

TypeScript is more syntax to learn and not the friendliest at times. It's also an additional build step so many devs prefer to stick with Javascript as long as JSDoc provides the same type-checking benefits.

> For the Type-Checked JavaScript option I'm still not sure what it means haha.

Quite a few people have gotten confused by it over on discord, and I've been meaning to try and make it clearer. Thank you for bringing it up.

I'm gonna resolve this so that people don't get confused by the outdated code snippet but if you have any further suggestions you can put them in the main thread. 🙂